internal/handlers/create_user_test.go

```diff
@@ -0,0 +1,70 @@
+package handlers_test
+
+import (
+	"bytes"
+	"github.com/golang/mock/gomock"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"go-api-backend/internal/handlers"
+	"go-api-backend/mocks"
+	"net/http"
+	"net/http/httptest"
+)
+
+var _ = Describe("CreateUserHandler", func() {
+
+	var userHandler *handlers.UserHandler
+	var mockRepo *mocks.MockUserRepositoryInterface
+
```

Comment:
Rather than declaring vars on each line you can do:
```go
var (
    userHandler *handlers.UserHandler
    mockRepo *mocks.MockUserRepositoryInterface
)
```